src/games/supported/AirRaid.cpp

```diff
@@ -101,3 +101,41 @@ ActionVect AirRaidSettings::getStartingActions() {
     startingActions.push_back(PLAYER_A_FIRE);
     return startingActions;
 }
+
+
+// returns a list of mode that the game can be played in
+ModeVect AirRaidSettings::getAvailableModes() {
+    ModeVect modes(getNumModes());
+    for (unsigned int i = 0; i < modes.size(); i++) {
+        modes[i] = i + 1;
+    }
+    return modes;
+}
+
+// set the mode of the game
+// the given mode must be one returned by the previous function
+void AirRaidSettings::setMode(game_mode_t m, System &system,
+                              std::unique_ptr<StellaEnvironmentWrapper> environment) {
+
+    if(m == 0) {
+        m = 1; // the default mode is not valid in this game
+    }
+    if(m >= 1 && m <= getNumModes()) {
+        //open the mode selection panel
+        environment->pressSelect(10);
```

Comment:
Is there a reason why this isn't pressSelect(20) ? Can you add a comment to this effect?

Reply:
I changed it to `pressSelect(20)`. To be honest, this is the unique game I was not able to validate. There's an issue in `ColourPalette`. I created an issue about this as well.